### ISSUE TITLE
Replication of the barely supervised learning experiments

### ISSUE THREAD

**Issue by jingshuangliu22** [google-research/fixmatch#57]
Hi, I am sorry to bother you but we met some problems to replicate the barely supervised learning experiments in your paper. 
I wonder if you could provide me with the **ordering of the CIFAR-10 training set** in this experiment. And my email address is 604475200@qq.com. Thanks a lot.
Looking forward to your reply.

**Comment by carlini** [comment]
I'm not entirely sure what you mean. We use the same data loader as provided in the code for the rest of the experiments, so the exact order of the training examples shouldn't (much) impact the training accuracy.

**Comment by jingshuangliu22** [comment]
I was sorry that I didn't describe my problem clearly. I am replicating the experiments in **_Section 4.4 Barely Supervised Learning_** in this paper (FixMatch: Simplifying Semi-Supervised Learning with Consistency and Confidence
). 
In this section, you claimed **_Specifically, we take the ordering of the CIFAR-10 training set from [5] that sorts examples from those that are most representative to those that are least._**
I have tried to google it but fail to get neither the **code** nor the **ordered CIFAR-10 training set** .
Consequently, I'd appreciate it a lot if you could offer me the ordered training set in this experiment.

**Comment by carlini** [comment]
Does #16 help answer your question?

**Comment by jingshuangliu22** [comment]
It's exactly what I needed. Appreciate it!  But since there is only one set, I still wondered if you could make the full ordered CIFAR-10 training set available. I would like to try to **randomly select the data** more times or **enlarge the training set** to verify this conclusion. Thanks a lot.

**Comment by carlini** [comment]
The ordering we computed in prior work is saved here:

https://github.com/tensorflow/privacy/releases/tag/0.2.3
